FateGrandAutomata/ScriptRunnerService.cs

```diff
@@ -89,6 +89,8 @@ public bool Stop()
             _virtualDisplay?.Release();
             _virtualDisplay = null;
 
+            _mediaProjection?.Stop();
+
```

Comment:
```suggestion
            _mediaProjection = null;

```

We'd also need to set the MediaProjection to `null` if we're doing this.
Otherwise, If the user clicks on `Toggle Service` again after stopping the service, the app will crash. We need to ask the user for the Media Projection permission again.
I've checked that AnkuLua also works in a similar way by asking the permission every time you start the service.